cdk/lib/discount-api.ts

```diff
@@ -125,9 +125,9 @@ export class DiscountApi extends GuStack {
 		const alarmDescription = (description: string) =>
 			`Impact - ${description}. Follow the process in https://docs.google.com/document/d/1_3El3cly9d7u_jPgTcRjLxmdG2e919zCLvmcFCLOYAk/edit`;
 
-		new GuAlarm(this, 'ApiGateway4XXAlarmCDK', {
+		new GuAlarm(this, 'ApiGateway5XXAlarmCDK', {
 			app,
-			alarmName: alarmName('API gateway 4XX response'),
+			alarmName: alarmName('API gateway 5XX response'),
 			alarmDescription: alarmDescription(
 				'Discount api received an invalid request',
```

Comment:
```suggestion
				'Discount api had something go wrong internally during a request',
```

Reply:
actually this is impact
```suggestion
				'Discount api could not display/apply a discount to an existing subscription via Manage',
```